### ISSUE TITLE
Why is the value loss multiplied by 0.5?

### ISSUE THREAD

**Issue by ZachariahRosenberg** [mimoralea/gdrl#10]
In the VPG implementation, the value loss is calculated,

`value_loss = value_error.pow(2).mul(0.5).mean()`

Isn't the value loss simply the MSE, so just `value_error.pow(2).mean()`? Why the additional multiplication of 0.5?

Thank you!

**Comment by ZachariahRosenberg** [comment]
Is this so that the "2" is removed from the derivative i.e. `f(x) = [(value_error) **2] / 2`, so therefore `f'(x) = value_error` ?
